src/components/SummaryDetail/SummaryDetail.js

```diff
@@ -10,32 +10,36 @@ const SummaryDetail = ({
   onOpen,
   title,
   renderTitle,
-}) => (
-  <div className={cx('slds-summary-detail', { 'slds-is-open': isOpen })}>
-    {onOpen && (
-      <IconButton
-        onClick={onOpen}
-        className={cx('slds-m-right_x-small', iconButtonClassName)}
-      >
-        <ButtonIcon
-          className="slds-summary-detail__action-icon"
-          icon="switch"
-          sprite="utility"
-        />
-      </IconButton>
-    )}
-    <div>
-      <div className="slds-summary-detail__title">
-        {renderTitle(title)}
-      </div>
-      {children && (
-        <div aria-hidden={!isOpen} className="slds-summary-detail__content">
-          {children({ isOpen })}
-        </div>
+}) => {
+  if (!onOpen) return renderTitle(title);
+
+  return (
+    <div className={cx('slds-summary-detail', { 'slds-is-open': isOpen })}>
+      {onOpen && (
+        <IconButton
+          onClick={onOpen}
+          className={cx('slds-m-right_x-small', iconButtonClassName)}
+        >
+          <ButtonIcon
+            className="slds-summary-detail__action-icon"
+            icon="switch"
+            sprite="utility"
+          />
+        </IconButton>
       )}
+      <div className="slds-container_fluid">
```

Comment:
I can't find this anywhere in the documentation on https://lightningdesignsystem.com/components/summary-detail. Where does this come from?

Reply:
<img width="673" alt="Screenshot 2020-01-17 at 11 16 28" src="https://user-images.githubusercontent.com/511551/72604353-04768d80-391b-11ea-8756-5724b8353d22.png">